IT079/BankSystemApplicationEntityFramework/BankSystemApplicationEntityFramework/Program.cs

```diff
@@ -0,0 +1,209 @@
+﻿using System;
+using System.Data;
+using Bank_App_Entity;
+
+class Program
+{
+    static void Main()
+    {
+        Console.WriteLine("Welcome to CDBank, India\n\n");
+        using (var context = new BankDbContext())
+        {
+            while (true)
+            {
+                Console.WriteLine("1. Create New User\n2. Update Existing User\n3. Delete Existing Account\n4. Add Money\n5. Withdraw Money\n6. Check Balance\n7. View All Customers\n8. Exit");
+
+                Console.WriteLine("\nEnter Your Choice: ");
+                int choice = int.Parse(Console.ReadLine());
+
+                if (choice == 1)
```

Comment:
Please use switch instead of if else ladder, Pls go through
https://medium.com/@michellekwong2/switch-vs-if-else-1d458e7b0711#:~:text=Some%20key%20advantages%20of%20switch%20over%20if%2Delse%20ladder%3A&text=It's%20because%20the%20compiler%20generates,compared%20to%20if%2Delse%20statements.